packages/core/src/modules/proofs/ProofsModule.ts

```diff
@@ -145,14 +154,59 @@ export class ProofsModule {
       requestedPredicates: proofRequestOptions.requestedPredicates,
     })
 
-    const { message, proofRecord } = await this.proofService.createRequest(connection, proofRequest, config)
+    const { message, proofRecord } = await this.proofService.createRequest(
+      { proofRequest, comment: proofRequestOptions.comment },
+      connection
+    )
 
     const outboundMessage = createOutboundMessage(connection, message)
     await this.messageSender.sendMessage(outboundMessage)
 
     return proofRecord
   }
 
+  /**
+   * Initiate a new presentation exchange as verifier by creating a presentation request
+   * not bound to any connection. The request must be delivered out-of-band to the holder
+   *
+   * @param proofRequestOptions Options to build the proof request
+   * @returns The proof record and proof request message
+   *
+   */
+  public async createOutOfBandRequest(proofRequestOptions: ProofRequestOptions): Promise<{
+    requestMessage: RequestPresentationMessage
+    proofRecord: ProofRecord
+  }> {
+    const nonce = proofRequestOptions.nonce ?? (await this.proofService.generateProofRequestNonce())
+
+    const proofRequest = new ProofRequest({
+      name: proofRequestOptions.name ?? 'proof-request',
+      version: proofRequestOptions.name ?? '1.0',
+      nonce,
+      requestedAttributes: proofRequestOptions.requestedAttributes,
+      requestedPredicates: proofRequestOptions.requestedPredicates,
+    })
```

Comment:
Shouldn't this be part of `proofService.createRequest` service? It seems like duplication from another method calling `proofService.createRequest`

Reply:
I didn't want to make opinions about default values in the service. I'm not even sure we should set default values...

Reply:
So for now I'd like to keep it in the module (and have a discussion about the duplication during the AFJ call)

Reply:
ok